CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog for AWS IoT Device Shadow library
 
+
+## v1.2.1 (November 2021)
```

Comment:
Update date here.

Reply:
Date and "MISRA compliance update" -> "MISRA C:2012 compliance update" update made.